### PR TITLE
Doc Doris sink

### LINKED ISSUE #1328: Document: feat(sink): support doris sink
This issue tracks the documentation update needed for the merged PR #12336.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/12336
Source PR Merged At: 2023-09-21T12:27:14Z

### PR DESCRIPTION
**Pull request by emile-00**
## Info

- **Description**

  Doc Doris sink, edit some typos in Clickhouse guide.

- **Notes**

  [ Include any supplementary context or references here. ]

- **Related code PR**

  https://github.com/risingwavelabs/risingwave/pull/12336

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1328

## For reviewers

- **Preview**

  [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **upcoming** version of the documentation. ]

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Comment by emile-00** [comment]
@xxhZs Is there a data type mapping table that we can include?

File: docs/guides/sink-to-doris.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+title: Sink data from RisingWave to Apache Doris
+description: Sink data from RisingWave to Apache Doris.
+slug: /sink-to-doris
+---
+
+This guide describes how to sink data from RisingWave to Apache Doris. Apache Doris is an open-source real-time data warehouse suitable for online analytical processing (OLAP). For more information, see [Apache Doris](https://doris.apache.org).
+
+## Prerequisites 
+
+- Ensure that RisingWave can access the network where the Doris BE and FE are located. For more details, see [Synchronize Data Through External Table](https://doris.apache.org/docs/dev/data-operate/import/import-scenes/external-table-load/).
```
Comment (hengm3467) [inline docs/guides/sink-to-doris.md]: Let's spell out what BE and FE stand for - backend and frontend I assume.

**Comment by xxhZs** [comment]
> @xxhZs Is there a data type mapping table that we can include?
 risingwave = > doris
 Boolean => BOOLEAN,
 smallint=> SMALLINT,
 integer=> INT,
 bigint=> BIGINT,
 real=>FLOAT
 double => DOUBLE
 Decimal =>DECIMAL
 Date => DATE
 Varchar =>STRING, VARCHAR
Time => not support
timestamp without time zone => DATETIME
timestamp with time zone=> not support
Interval =>not support
Struct => STRUCT
array => ARRAY
Bytea => not support
Jsonb => JSONB,
Serial => BIGINT

**Review by hengm3467 — APPROVED** [review]
LGTM. Thanks!

**Comment by hengm3467** [comment]
@emile-00 Please make sure to include the data mapping table in the topic. Thanks.

**Comment by xxhZs** [comment]
> > @xxhZs Is there a data type mapping table that we can include?
> > risingwave = > doris
> > Boolean => BOOLEAN,
> > smallint=> SMALLINT,
> > integer=> INT,
> > bigint=> BIGINT,
> > real=>FLOAT
> > double => DOUBLE
> > Decimal =>DECIMAL
> > Date => DATE
> > Varchar =>STRING, VARCHAR
> > Time => not support
> > timestamp without time zone => DATETIME
> > timestamp with time zone=> not support
> > Interval =>not support
> > Struct => STRUCT
> > array => ARRAY
> > Bytea => not support
> > Jsonb => JSONB,
> > Serial => BIGINT


Here, it may be necessary to emphasize the length and precision issues of decimal, similar to：
"1. RW will round to the nearest to ensure precision matches that of Doris. 2. Please ensure that the length of decimal types being imported into Doris does not exceed Doris's decimal length, otherwise it will fail to import."

**Review by xxhZs — COMMENTED** [review]
LGTM

**Review by xxhZs — APPROVED** [review]

File: docs/guides/sink-to-doris.md (hunk 1 above)
Comment (emile-00) [reply docs/guides/sink-to-doris.md]: @xxhZs Can you help confirm if BE and FE stand for backend and frontend? Thanks!

Comment (hengm3467) [reply docs/guides/sink-to-doris.md]: @emile-00 Yes. They are. Please see this page:

https://doris.apache.org/docs/1.2/admin-manual/config/fe-config/

**Pull request merged**